android/app/src/main/assets/shell-app-manifest.json

```diff
@@ -37,6 +37,10 @@
   "orientation": "portrait",
```

Comment:
does this stuff get generated/updated when you `expo publish`?

Reply:
It should.

Reply:
haha i guess we'll just commit this all the time then 👍